src/lib/input/autosize.ts

```diff
@@ -17,9 +17,25 @@ export class MdTextareaAutosize implements OnInit {
   /** Minimum number of rows for this textarea. */
   @Input() minRows: number;
 
+  get mdAutosizeMinRows(): number {
+    return this.minRows;
+  }
+
+  @Input() set mdAutosizeMinRows(value: number) {
```

Comment:
`@Input()` should go on the `getter`. We should add that to the coding standards...